tensorflow/python/distribute/multi_process_runner.py

```diff
@@ -318,7 +318,13 @@ def _start_subprocess_and_reading_thread(self,
         target=_ProcFunc(),
         args=(resources, test_env, fn, args, kwargs, self._use_dill_for_args),
         daemon=self._daemon)
+    logging.info(f"[{task_type}-{task_id}] 
+                    Starting subprocess (PID will be assigned)")
```

Comment:
```suggestion
    logging.info(f"[{task_type}-{task_id}] "
                 "Starting subprocess (PID will be assigned)")
```

Reply:
You cannot split a string between two lines, instead you have to rely on the implicit concatenation feature.

Reply:
Also, for logging please don't use `f` string, use the `%` syntax instead. This is because f-strings get evaluated at call site, whereas the `%` syntax evaluates the arguments __only__ if logging is enabled.